src/lib/storage/disk_analyzer.rb

```diff
@@ -286,10 +286,14 @@ def installation_disk?(disk)
       # minor/major IDs since the inst-sys is in a RAM disk (copied from the
       # installation medium).
       #
-      def installation_volume?(vol)
-        log.info("Checking if #{vol.name} is an installation volume")
-        is_inst = mount_and_check(vol) { |mp| installation_volume_check(mp) }
-        log.info("#{vol} is installation medium") if is_inst
+      # @param vol_name [string] device name of the volume to check
+      #
+      # @return [bool] 'true' if the volume is an installation volume
+      #
+      def installation_volume?(vol_name)
+        log.info("Checking if #{vol_name} is an installation volume")
+        is_inst = mount_and_check(vol_name) { |mp| installation_volume_check(mp) }
+        log.info("#{vol_name} is installation medium") if is_inst
```

Comment:
I'd always log the result here. If you log "Checking if FOO is an installation volume", there should be always also the result.

Otherwise you have to check the source code to find out that if the result message is missing it is not an installation volume. Which is quite tricky...